src/lib.rs

```diff
@@ -17,6 +17,7 @@ pub mod body;
 mod endpoint;
 mod extract;
 pub mod head;
+mod logger;
```

Comment:
We should consider putting this into a submodule of `middlware` and exposing it publicly that way.